src/plugins/columnSummary/endpoints.js

```diff
@@ -338,10 +331,10 @@ class Endpoints {
       const newRange = [];
       if (range[1]) {
         for (let i = range[0]; i <= range[1]; i++) {
-          newRange.push(this.recordTranslator.toPhysicalRow(i));
+          newRange.push(this.plugin.rowIndexMapper.getPhysicalIndex(i));
```

Comment:
This class have private `getVisualRowIndex`, `getVisualColumnIndex` that use `this.hot.toVisualRow` public method. Which is good because we use the same public API our users does.